hs_access_control/models/user.py

```diff
@@ -2077,9 +2077,13 @@ def can_undo_share_group_with_user(self, this_group, this_user):
         if not self.user.is_active:
             raise PermissionDenied("Requesting user is not active")
```

Comment:
@alvacouch Should line 2078 be changed to return False rather than raise PermissionDenied exception as well?

Reply:
The PR is to eliminate a very specific behavior: user to resource sharing. That bug doesn't occur when sharing groups with users, to my knowledge. I'm willing to change it but it wouldn't fix any bug I know about.